README.md

```diff
@@ -39,7 +39,7 @@
 1. Open [localhost:8000](http://localhost:8000) in a browser (I recommend Firefox and Chrome).
 1. Modify source files (e.g. `/src/lib.rs` or `/index.html`).
 1. Watch compilation in the terminal tab where you run `cargo make watch`.
-1. You can watch dev-server responses in the tab where you run `cargo make server`.
+1. You can watch dev-server responses in the tab where you run `cargo make serve`.

```

Comment:
It's not a typo - see https://github.com/seed-rs/seed-quickstart/blob/c7a47bf871293b912ae9ca49ab47c21a0ddedbad/Makefile.toml#L16